pkg/controller/pattern1/deployment.go

```diff
@@ -195,7 +199,7 @@ func Apim2Deployment(apimanager *apimv1alpha1.APIManager, z *configvalues, num i
 	initContainers = append(initContainers, apim1InitContainer)
 
 	// appending the analytics-worker init container
-	initContainers = append(initContainers, getAnalyticsWorkerInitContainers())
+	//initContainers = append(initContainers, getAnalyticsWorkerInitContainers())
```

Comment:
Can we remove this code segment if we don't need this in the future?

Reply:
This line doesn't exist in the master branch

Reply:
Closing since the mentioned line is not in the master branch